Market Maker Guide.md

```diff
@@ -300,27 +300,28 @@ Balance:
 
 ## Connect Arby
 
-In this final step we are connecting your xud setup to your Binance account via a tool called ["arby"](https://github.com/ExchangeUnion/market-maker-tools). This enables "transfer" of orders from Binance into OpenDEX and creates an arbitrage revenue stream for you as market maker. Arby issues orders on OpenDEX based on the Binance price adding a `margin` as premium. When orders are filled on OpenDEX, arby takes care of executing a counter trade on Binance to lock in profits. You will need funds for at least one supported asset on Binance (e.g. BTC) for arby to start issuing orders. To activate arby, `exit` from `xud ctl` and run `cp ~/.xud-docker/mainnet/sample-mainnet.conf ~/.xud-docker/mainnet/mainnet.conf` to create a config file for your environment. Then edit the following options in `mainnet.conf`:
+In this final step we are connecting your xud setup to your CEX (Centralized EXchage) account via a market maker bot called ["arby"](https://github.com/ExchangeUnion/market-maker-bot). Arby enables "transfer" of orders from the CEX into OpenDEX and creates an arbitrage revenue stream for you as market maker. Arby issues orders on OpenDEX based on the CEX price, adding a `margin` as premium. When orders are filled on OpenDEX, arby takes care of executing a counter trade on the CEX to lock in profits.At the time of writing, arby supports connecting to Binance and Kraken, but more exchanges will be added over time - check the [FAQs](https://github.com/ExchangeUnion/market-maker-bot#faq) for an up-to-date list. We'll use Binance in this example. You will need funds for at least one supported asset on Binance (e.g. BTC) for arby to start issuing orders. To activate arby, `exit` from `xud ctl` and run `cp ~/.xud-docker/mainnet/sample-mainnet.conf ~/.xud-docker/mainnet/mainnet.conf` to create a config file for your environment. Then edit the following options in `mainnet.conf`:
 ```bash
 xud@ubuntu:~$ nano ~/.xud-docker/mainnet/mainnet.conf
 # this option needs to be set to true to allow arby to execute Binance orders on your behalf, needed for arby to function
```

Comment:
add `[arby]` in first row, otherwise config will not work

Reply:
The instructions are based on copying the sample config file and `[arby]` is already in there, anyhow - added a hint that this is the place where the arby config section starts.